### DISCUSSION TITLE
Help with CSS and Statblock Coloring

### DISCUSSION THREAD

**Discussion by cenycal** [Obsidian-TTRPG-Community/fantasy-statblocks#73] [Q&A] [answered]
So I have no CSS skills. So while I see the CSS Color Code Block in the Readme, I don't understand where I should be looking to edit the colors of statblocks on a global level.

Can I get some more specific instructions on where to either find that code to edit the colors or how to include color specifics in a custom layout?

**Comment by FilBot3** [comment]
I do agree that the README needs a bit more explanation on the various Custom Layout Blocks and the CSS that is available to change or how to do it.

**Comment by FilBot3** [comment]
I would like to see how I can adjust the column width so the text can take up the whole Readable Text area. Even with 2 columns, the text seems limited to 80 chars per line.

**Reply by kylomax** [reply, marked as answer]
You can adjust this without adjusting the CSS. In the statblock code block, just add this:
```
columnWidth: <width, default is 400> 
forceColumns: true
```

**Reply by FilBot3** [reply]
This did the trick for me! Thank you!
